lib/versioning/loose/generic.ts

```diff
@@ -191,10 +192,11 @@ export abstract class GenericVersioningApi<
 
   // eslint-disable-next-line class-methods-use-this
   getNewValue(
-    _currentValue: string,
-    _rangeStrategy: RangeStrategy,
-    _fromVersion: string,
-    toVersion: string
+    { toVersion }: NewValueConfig = {
+      currentValue: null,
+      rangeStrategy: null,
+      toVersion: arguments[0],
+    }
   ): string {
     return toVersion;
   }
```

Comment:
@rarkins I don't think, this as a good approach. Any idea to make it better?

Reply:
Can we un-expand the params and just make it like `getNewValue(newValueConfig: NewValueConfig)` and then `return newValueConfig.toVersion` ?

Reply:
> That looks very strange to me. Why we need `arguments[0]`?

If the value is falsy then it cannot be destructured. The test was expecting the same value as it is expecting.

Reply:
> Can we un-expand the params and just make it like `getNewValue(newValueConfig: NewValueConfig)` and then `return newValueConfig.toVersion` ?

What if it is a falsy value? `return` will break there.

Reply:
then we should simply use `return arg?.toVersion`  <-- New typescript null check